src/lib/db.ts

```
@@ -1,7 +1,24 @@
+// src/lib/db.ts
 import { PrismaClient } from "@prisma/client";
 
-const globalForPrisma = global as unknown as { prisma: PrismaClient };
+const prismaClientSingleton = () => {
+  return new PrismaClient();
+};
 
-export const prisma = globalForPrisma.prisma || new PrismaClient();
+declare const globalThis: {
```

Biome (2.1.2)

[error] Do not shadow the global "globalThis" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)